hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -3395,7 +3394,7 @@ public void testLocalDevice() {
         assertEquals(blockSize * device0Multiplier, localDeviceConfig.getBlockSize());
         assertEquals(readIOThreadCount * device0Multiplier, localDeviceConfig.getReadIOThreadCount());
         assertEquals(writeIOThreadCount * device0Multiplier, localDeviceConfig.getWriteIOThreadCount());
-        assertEquals(new MemorySize(1234567890, MemoryUnit.MEGABYTES), localDeviceConfig.getCapacity());
+        assertEquals(new Capacity(1234567890, MemoryUnit.MEGABYTES), localDeviceConfig.getCapacity());
```

Comment:
Should we have a test similar to `testNativeMemoryConfiguration_isBackwardCompatible` here as well?